orc-rt/include/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(ORC_RT_HEADERS
+    orc-rt-c/orc-rt.h
+)
+
+# TODO: Switch to filesets when we move to cmake-3.23.
+add_library(orc-rt-headers INTERFACE)
+target_include_directories(orc-rt-headers INTERFACE
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+)
+set_property(TARGET orc-rt-headers
+    PROPERTY PUBLIC_HEADER ${ORC_RT_HEADERS}
+)
+install(DIRECTORY ./
```

Comment:
I'd prefer using a separate `install` rule for each header in `ORC_RT_HEADERS` rather than a directory rule. This is is more explicit and matches what we use elsewhere in LLVM.

Reply:
Note that using `FILE_SET` specification makes it a lot easier to manage this, but that is CMake 3.23+.